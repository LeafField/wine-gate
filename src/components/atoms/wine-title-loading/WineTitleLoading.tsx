import React from "react";

const WineTitleLoading = () => {
  return (
    <div className="h-[4.375rem] max-w-[64.9375rem] animate-pulse bg-gray tablet:w-[calc(var(--vw)-401px)]">
      <span className="sr-only">読み込み中...</span>
    </div>
  );
};

export default WineTitleLoading;
