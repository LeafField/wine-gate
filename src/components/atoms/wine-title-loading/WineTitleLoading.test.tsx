import { screen, render } from "@testing-library/react";
import WineTitleLoading from "./WineTitleLoading";

describe("WineTitleLoadingの単体テスト", () => {
  test("スナップショットテスト", () => {
    const { container } = render(<WineTitleLoading />);
    expect(container).toMatchSnapshot();
  });
});
