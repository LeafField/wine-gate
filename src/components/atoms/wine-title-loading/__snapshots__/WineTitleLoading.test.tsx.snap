// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WineTitleLoadingの単体テスト スナップショットテスト 1`] = `
<div>
  <div
    class="h-[4.375rem] max-w-[64.9375rem] animate-pulse bg-gray tablet:w-[calc(var(--vw)-401px)]"
  >
    <span
      class="sr-only"
    >
      読み込み中...
    </span>
  </div>
</div>
`;
